src/Components/FormCreation/CardComponents/TextQuestion.tsx

```diff
@@ -96,19 +98,23 @@ function TextQuestion({
   };
 
   const [isValid, setIsValid] = useState(
-    !initialValidation || initialValidation?.min === 0
+    initialValidation && isTextValid(initialValidation, initialAnswer)
```

Comment:
Remember that textQuestion is also used for Form Creation
```suggestion
    submission && initialValidation && isTextValid(initialValidation, initialAnswer)
```

You need to check to see if it is a submission